packages/strapi-plugin-users-permissions/config/schema.graphql.js

```diff
@@ -57,8 +57,8 @@ module.exports = {
   mutation: `
     login(input: UsersPermissionsLoginInput!): UsersPermissionsLoginPayload!
     register(input: UserInput!): UsersPermissionsLoginPayload!
-    forgotPassword(email: String!): ForgotPassword
-    changePassword(password: String!, passwordConfirmation: String!, code: String!): UsersPermissionsLoginPayload
+    forgotPassword(email: String!): UserPersmissionsPasswordPayload
```

Comment:
Hey there's a spelling mistake here User**Persmissions**PasswordPayload** it should be User**Permissions**PasswordPayload

Reply:
nice catch, at least the code works :) Mind opening a PR ?

Reply:
nice catch, Sanoj, hope u don't mind I had it open, so I just pushed the patch :) https://github.com/strapi/strapi/pull/6390